File: Jon Agirre/PROGRAMAZIOA/Erronka_2/Bluej_ikusteko/Proba1/Kanta.java
public class Kanta extends Diskoak
{   
    private String izena;
    private String egilea;
    private String urtea;
    private int iraupena;
    private int zenbakia;
    public Kanta()
    {
        
    }
    public Kanta(int z,String a,String b,String c,int d)
    {   zenbakia = z;
        izena = a;
        egilea = b;
        urtea = c;
        iraupena = d;
    }
    public String getIzena()
    {   return izena;
    }
    public String getEgilea()
    {   return egilea;
    }
    public String getUrtea()
    {   return urtea;
    }
    public int getIraupena()
    {   return iraupena;
    }
    public void MenuanErakutsi()
    {   System.out.printf("%d. %s - %s (%s)\n",zenbakia, izena, egilea, urtea);
    }
    public void setEntzuten()
    {   System.out.printf("\nOrain entzuten: %s - %s (%s)\n\n",izena, egilea, urtea);
    }
}

File: Jon Agirre/PROGRAMAZIOA/Erronka_2/Python/Kanta.py
class Kanta:
    def __init__(Kanta,z,a,b,c,d):
        Kanta.izena = a
        Kanta.egilea = b
        Kanta.urtea = c
        Kanta.iraupena = d
        Kanta.zenbakia = z
    
    def getZenbakia():
        return Kanta.zenbakia
    def getIzena(Kanta):
        return Kanta.izena
    
    def MenuanErakutsi(Kanta):
        print(f'{Kanta.zenbakia}. {Kanta.izena} - {Kanta.egilea} ({Kanta.urtea})\n')
    
    def Entzuten(Kanta):
        print(f'\nOrain entzuten: {Kanta.izena} - {Kanta.egilea} ({Kanta.urtea})')